src/travel-aid/TravelAidScreenComponent.tsx

```diff
@@ -114,7 +121,21 @@ const TravelAidSection = ({
     fromQuayId,
   );
 
-  useTravelAidAnnouncements({status, focusedEstimatedCall});
+  let notices: NoticeFragment[] = [];
+  if (serviceJourney !== undefined) {
+    notices = getNoticesForServiceJourney(serviceJourney, fromQuayId) ?? [];
+  }
+
+  const situations =
+    focusedEstimatedCall?.situations.sort((n1, n2) =>
+      n1.id.localeCompare(n2.id),
+    ) ?? [];
```

Comment:
`focusedEstimatedCall` shouldn't ever be undefined

```suggestion
  const situations = focusedEstimatedCall.situations.sort((n1, n2) =>
    n1.id.localeCompare(n2.id),
  );
```